cmd/manager/aggregator.go

```diff
@@ -322,6 +322,10 @@ func getCheckResultLabels(pr *utils.ParseResult, resultLabels map[string]string,
 	labels[compv1alpha1.SuiteLabel] = scan.Labels[compv1alpha1.SuiteLabel]
 	labels[compv1alpha1.ComplianceCheckResultStatusLabel] = string(pr.CheckResult.Status)
 	labels[compv1alpha1.ComplianceCheckResultSeverityLabel] = string(pr.CheckResult.Severity)
+	if len(pr.CheckResult.ValueUsed) > 0 {
+		labels[compv1alpha1.ComplianceCheckResultValueLabel] = ""
+	}
```

Comment:
I'm not sure a label is needed for this if we're adding the `ValuesUsed` key to the CRD itself. Is there any specific use-case in mind for adding this label?

Reply:
That's a good idea, I was testing if it has worked since the `ValuesUsed` did not show up, I will deleted it!